__tests__/lib/babel.js

```diff
@@ -1,11 +1,13 @@
 /* globals describe, expect, it */
 
+const INTENDED_LENGTH = 2
```

Comment:
Seems like this should be defined within the test case.  The intended length could be different in other test cases/suites.